parity/cli/mod.rs

```diff
@@ -728,6 +728,10 @@ usage! {
 			"--gas-price-percentile=[PCT]",
 			"Set PCT percentile gas price value from last 100 blocks as default gas price when sending transactions.",
 
+			ARG arg_poll_lifetime: (u32) = 60u32, or |c: &Config| c.mining.as_ref()?.poll_lifetime.clone(),
+			"--poll-lifetime=[S]",
+			"Set the lifetime of the internal index filter to S seconds. Deafult: 60",
```

Comment:
Didn't review; just a nitpick: you don't need to specify the default value in the help message; it's automatically appended.